setup.cfg

```diff
@@ -65,6 +65,7 @@ jupyter = notebook; ipywidgets>=7.0
 dev =
     black ; python_version>='3.6'
     coverage
+    czml3
```

Comment:
Hmmm let's add it under `install_requires` instead until we sort https://github.com/poliastro/poliastro/issues/168#issuecomment-421963430 out